src/styles/header.scss

```diff
@@ -0,0 +1,23 @@
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
```

Comment:
will be better to move these styles to `normalize.css` and `reset.css` files
They don't relate to header block